pkg/preparer/orchestrate.go

```diff
@@ -79,6 +79,90 @@ func (p podWorkerID) String() string {
 	return fmt.Sprintf("%s-%s", p.podID.String(), p.podUniqueKey)
 }
 
+func (p *Preparer) InstallWhiteListPods(whiteListPods map[types.PodID]bool, preparerConfig *PreparerConfig) error {
+	intentResults, _, err := p.store.ListPods(consul.INTENT_TREE, p.node)
+	realityResults, _, err := p.store.ListPods(consul.REALITY_TREE, p.node)
+	var intents []consul.ManifestResult
+	if err != nil {
+		p.Logger.WithError(err).Errorln("could not check intent")
+		return err
+	}
+	errorChan := make(chan error)
+	quit := make(chan struct{})
+
+	for _, intentResult := range intentResults {
+		intentSHA, _ := intentResult.Manifest.SHA()
+		if _, ok := whiteListPods[intentResult.Manifest.ID()]; ok {
```

Comment:
From https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow, you could reduce indentation with:
```
ok := whiteListPods[intentResult.Manifest.ID()]
if !ok {
        break
} 
// rest of loop body, now indented one less level.
```
This idea is shown in the fourth example of the link above.

Reply:
Different cases, I'm checking a map, not related to the error flow in your link. 
Same usage in check.go line 345

Reply:
It is Go convention to reduce the indentation level of the normal path. The [counterexample](https://github.com/square/p2/blob/master/integration/single-node-slug-deploy/check.go#L345) you provide is not the same. It checks for `!ok`, whereas you check for `ok`. Notice how the `!ok` check there is an exceptional, error returning case; it is not the normal path.

Here, you are unnecessarily nesting the normal path within an if block.

Reply:
The counterexample is the same case here to check the existence of a key in map. Well, as the value here is bool already, I would remove "ok".